plugin/tmux_navigator.vim

```diff
@@ -78,7 +78,7 @@ function! s:TmuxAwareNavigate(direction)
       catch /^Vim\%((\a\+)\)\=:E141/ " catches the no file name error 
       endtry
     endif
-    let args = 'select-pane -t ' . $TMUX_PANE . ' -' . tr(a:direction, 'phjkl', 'lLDUR')
+    let args = 'select-pane -t \' . $TMUX_PANE . ' -' . tr(a:direction, 'phjkl', 'lLDUR')
```

Comment:
Good call. It seems that enclosing in quotes works as well as escaping with `\`.